### DISCUSSION TITLE
RS41 fail to connect the programmer

### DISCUSSION THREAD

**Discussion by pimol** [mikaelnousiainen/RS41ng#59] [Q&A] [answered]
One of my RS41s does not connect to the programmer (ST2 USB) after installing a small test software. I have 3 others, all work fine. Is there any trick to recover it or is it broken for sure?

Pietro

**Comment by iz2fly** [comment]
I don't know what to suggest, maybe recheck the electrical connections, verify that the memory has been completely erased.
It only happened to me once, the sonde was crushed by a car, but it was programmed with ST-Link V2, but the signal was not always good.

**Comment by pimol** [comment]
Double checked the connection. I swapped 2 or 3 times between this and
all my  good ones. It's the only one that the programmer doesn't find, so I
can't check any value in the EPROM. I think she's fucked, but I'd like to
understand why...

Pietro

Il giorno sab 23 set 2023 alle ore 19:31 iz2fly ***@***.***>
ha scritto:

**Reply by pimol** [reply, marked as answer]
Solved. I take the RS41 under reset, then try connect and I release the reset: connected both the board that was broken.
